CONTRIBUTING.md

```diff
@@ -2,44 +2,58 @@
 
 Hi! Nice to have you with us. This file will walk you through the steps you need to take if you want to contribute a word or some code to the project. Let’s go.
 
-Actually – before we start – a quick word of caution: Unfortunately the contribution process involves some technical knowledge. We aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
+Actually – before we start – a quick word of caution: unfortunately, the contribution process involves some technical knowledge. We are aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
 
 ## Contributing Words
 
 ### Version 1: Issues
 
-If you know a word that should be defined, but – for whatever reason – don’t feel confident writing the definition yourself (or maybe you disagree with an existing defintion), please open an issue. We can then collectively figure out how to best get this word defined.
+If you know a word that should be defined, but – for whatever reason – don’t feel confident writing the definition yourself (or maybe you disagree with an existing defintion), please open an [issue](https://github.com/tatianamac/selfdefined/issues/new). We can then collectively figure out how to best get this word defined.
```

Comment:
This is preference but could we please ensure all emdashes do not have spaces around them 🙈

Reply:
Oh wow, I've just learned something new and counter-intuitive about English punctuation! I'll remove them, sure.

Reply:
If you want to spend to much time on punctuation you would a thin space on either side of the em dash to avoid that the dash feels to close to the characters (m—m vs m — m). While that’s very valuable trivia to impress … mostly nobody with, it is completely useless knowledge in everyday life. 

Actually, looking closer at it, @tatianamac, the spaces in use here are en dashes (–) and as such need to be spaced. Or changed to em dashes (—).

Excuse us, Ekaterina, typographers can be incredibly annoying. 🙈

Reply:
I did change them to em dashes.

@ovlb It's fine. If the text would've been in Russian, you'd need to put spaces there. Makes you think how arbitrary all these rules are.

Reply:
In German you would use en dashes with a normal space. Fun, fun, fun.